src/test/java/org/broadinstitute/consent/http/mail/message/DaaRequestMessageTest.java

```diff
@@ -0,0 +1,79 @@
+package org.broadinstitute.consent.http.mail.message;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import freemarker.template.Template;
+import java.io.StringWriter;
+import java.io.Writer;
+import java.util.Objects;
+import org.broadinstitute.consent.http.AbstractTestHelper;
+import org.broadinstitute.consent.http.configurations.FreeMarkerConfiguration;
+import org.broadinstitute.consent.http.mail.freemarker.FreeMarkerTemplateHelper;
+import org.broadinstitute.consent.http.models.User;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class DaaRequestMessageTest extends AbstractTestHelper {
+
+  private FreeMarkerTemplateHelper helper;
+
+  @BeforeEach
+  void setUp() {
+    FreeMarkerConfiguration freeMarkerConfig = new FreeMarkerConfiguration();
```

Comment:
These tests ended up having some common code. A shared base class could be added to avoid this duplication.